src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -2928,6 +2747,13 @@ public void var4(InputType inputType, ReaderType readerType, TransformType trans
         }
     }
 
+    internal sealed class AllowDefaultResolverContext : IDisposable
```

Comment:
1. I think this should set value to previous
2. Please ensure tests using this do not run in parallel (xunit attribute with a comment why)